_includes/configuration/rtc_encryption_provider.md

```diff
@@ -0,0 +1,53 @@
+## `rtc_encryption_provider`
+
+The RTC plugin requires an encryption key for end-to-end encryption. This key is not sent to the {{site.cloudname}} server; the {{site.productname}} RTC service cannot read the editor content. The encryption key is used by the [SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto) browser API to encrypt and decrypt the editor content in the user's browser.
```

Comment:
```suggestion
The RTC plugin requires an encryption key for end-to-end encryption. This key is not sent to the RTC server; the {{site.productname}} RTC service cannot read the editor content. The encryption key is used by the [SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto) browser API to encrypt and decrypt the editor content in the user's browser.
```
Again, this won't be accurate once self-hosted is available so removing the cloud specific part.